src/common/reducers/repositories.js

```diff
@@ -2,10 +2,11 @@ import * as ActionTypes from '../actions/repositories';
 import { parseGitHubRepoUrl } from '../helpers/github-url';
 
 export function repositories(state = {
-  isFetching: false,
+  // has no server side content, so initial state is always 'fetching'
+  isFetching: true,
```

Comment:
Ok, so I went round in my head about this a few times. Here's the argument for what I've done:

The component in it's initial state on mount has no data, so it's 'in need of data' and should show a spinner to let the user know that; calling `fetch()` is just a consequence of the state of being without data, so the property might be better as `needsData` perhaps to break the semantic from `fetch` API to component state.

Does that make sense? And if it makes sense is it valid?